featurebyte/routes/common/schema.py

```diff
@@ -11,6 +11,10 @@
 # route query parameters
 COLUMN_STR_MAX_LENGTH = 255
 COLUMN_STR_MIN_LENGTH = 1
+PREVIEW_SEED = 1234
+PREVIEW_LIMIT = 10000
+PREVIEW_DEFAULT = 10
+DESCRIPTION_SIZE_LIMIT = 1000000
```

Comment:
nit: would be good to have thousands delimiters to make this a bit more readable.